broker/src/main/java/io/zeebe/broker/system/SystemContext.java

```diff
@@ -76,13 +75,7 @@ private void validateConfiguration() {
       throw new IllegalArgumentException(String.format(NODE_ID_ERROR_MSG, nodeId, clusterSize));
     }
 
-    final StorageLevel storageLevel = data.getAtomixStorageLevel();
     final int replicationFactor = cluster.getReplicationFactor();
-
-    if (storageLevel == StorageLevel.MAPPED && replicationFactor > 1) {
-      throw new IllegalStateException(MMAP_REPLICATION_ERROR_MSG);
```

Comment:
I guess we can also remove the constant `MMAP_REPLICATION_ERROR_MSG`?

Reply:
Woops, I'll make sure to remove it.